<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Archiver</string>
  </property>
  <property name="windowIcon">
   <iconset theme="lxqt-archiver">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="DirTreeView" name="dirTreeView">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="FileTreeView" name="fileListView">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="filterLineEdit">
          <property name="placeholderText">
           <string>Filter Files...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCreateNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExtract"/>
    <addaction name="actionTest"/>
    <addaction name="separator"/>
    <addaction name="actionArchiveProperties"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionRename"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
    <addaction name="separator"/>
    <addaction name="actionAddFiles"/>
    <addaction name="actionAddFolder"/>
    <addaction name="separator"/>
    <addaction name="actionPassword"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuIcon_Size">
     <property name="title">
      <string>&amp;Icon Size</string>
     </property>
     <addaction name="action16px"/>
     <addaction name="action24px"/>
     <addaction name="action32px"/>
     <addaction name="action48px"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionToolbar"/>
    <addaction name="actionStatusbar"/>
    <addaction name="actionDirTree"/>
    <addaction name="separator"/>
    <addaction name="menuIcon_Size"/>
    <addaction name="actionFlatListMode"/>
    <addaction name="actionDirTreeMode"/>
    <addaction name="separator"/>
    <addaction name="actionExpand"/>
    <addaction name="actionCollapse"/>
    <addaction name="separator"/>
    <addaction name="actionStop"/>
    <addaction name="actionReload"/>
    <addaction name="separator"/>
    <addaction name="actionFilter"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreateNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExtract"/>
   <addaction name="separator"/>
   <addaction name="actionAddFiles"/>
   <addaction name="actionAddFolder"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionCreateNew">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="icon">
    <iconset theme="archive-extract">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Extract</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>&amp;Test</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionArchiveProperties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Properties</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="window-close">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAddFiles">
   <property name="icon">
    <iconset theme="archive-insert">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Files</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset theme="archive-insert-directory">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Add F&amp;older</string>
   </property>
  </action>
  <action name="actionPassword">
   <property name="text">
    <string>Pass&amp;word</string>
   </property>
  </action>
  <action name="actionToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Toolbar</string>
   </property>
  </action>
  <action name="actionStatusbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Statusbar</string>
   </property>
  </action>
  <action name="actionDirTree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Directory Tree</string>
   </property>
  </action>
  <action name="actionDirTreeMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>S&amp;how as Folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionFlatListMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show &amp;All Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="process-stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>St&amp;op</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>&amp;Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionFilenameEncoding">
   <property name="text">
    <string>Filename &amp;Encoding</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>&amp;View Selected Items</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="text">
    <string>&amp;Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="icon">
    <iconset theme="expand-all"/>
   </property>
   <property name="text">
    <string>&amp;Expand</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionCollapse">
   <property name="icon">
    <iconset theme="collapse-all"/>
   </property>
   <property name="text">
    <string>&amp;Collapse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="action16px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>16 px</string>
   </property>
  </action>
  <action name="action24px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>24 px</string>
   </property>
  </action>
  <action name="action32px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>32 px</string>
   </property>
  </action>
  <action name="action48px">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>48 px</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileTreeView</class>
   <extends>QTreeview</extends>
   <header>filetreeView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DirTreeView</class>
   <extends>QTreeview</extends>
   <header>dirtreeView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStatusbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>statusBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToolbar</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
